components/server/src/ome/security/SecuritySystem.java

```diff
@@ -119,10 +114,8 @@
     /**
      * Prepares the current {@link EventContext} instance with the current
      * {@link Principal}. An exception is thrown if there is none.
-     * 
-     * @param isReadOnly
      */
-    void loadEventContext(boolean isReadOnly);
```

Comment:
Do you want to open a 5.3 PR and mark it as deprecated?
so we can/cannot remove it later on depending on outcome